spring-rabbit/src/test/java/org/springframework/amqp/rabbit/annotation/EnableRabbitIntegrationTests.java

```diff
@@ -306,6 +320,56 @@ public RabbitAdmin rabbitAdmin(ConnectionFactory connectionFactory) {
 			return new RabbitAdmin(connectionFactory);
 		}
 
+		@Bean
+		public MultiListenerBean multiListener() {
+			return new MultiListenerBean();
+		}
+
+	}
+
+	@RabbitListener(bindings = @QueueBinding
+			(value = @Queue,
```

Comment:
True (default). I didn't think it was worth complicating the BPP with `AnonymousQueue` since it's just a subclass with specific settings - handled in the BPP.